app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,37 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+    </style>
+
+    <!-- Override Yoti fonts style to use GothicA1 -->
+    <style name="Yoti.v3.Font.DisplayBold">
```

Comment:
We also have to correct the font overriding here. The fonts that can be overridded are: 
- YotiDocs.TextView.Title
- YotiDocs.TextView.Header
- YotiDocs.TextView.Text
- YotiDocs.TextView.Link